src/test/java/com/sorts/CombSortTest.java

```diff
@@ -0,0 +1,85 @@
+package com.sorts;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CombSortTest {
+
+    //One global combSort instance variable
```

Comment:
The comment can be removed, not really needed.

Reply:
done.